index.html

```diff
@@ -37,11 +37,11 @@ <h1>I’m Tuyishimire Eric Glad to see you!</h1>
                     don’t hestiate to contact me.</p>
                 <p class="connect">LET'S CONNECT</p>
                 <ul class="social-1">
-                    <li><a href="index.html"><img src="image/1.png" alt="social1"/></a></li>
-                    <li><a href="index.html"><img src="image/2.png" alt="social2"/></a></li>
-                    <li><a href="index.html"><img src="image/3.png" alt="social3"/></a></li>
-                    <li><a href="index.html"><img src="image/4.png" alt="social4"/></a></li>
-                    <li><a href="index.html"><img src="image/5.png" alt="social5"/></a></li>
+                    <li><a href="index.html"><img src="image/1.png" alt="Twitter"/></a></li>
+                    <li><a href="index.html"><img src="image/2.png" alt="Linkedin"/></a></li>
+                    <li><a href="index.html"><img src="image/3.png" alt="Medium"/></a></li>
+                    <li><a href="index.html"><img src="image/4.png" alt="Github"/></a></li>
+                    <li><a href="index.html"><img src="image/5.png" alt="Angellist"/></a></li>
```

Comment:
We identified that it would be better to name the alternative texts in social media images with related names. For example we suggested the twitter icon to have an identifiable name in the alt text. We named it as "twitter". This applied to other images as well to make it readable.